### DISCUSSION TITLE
Can't parse PPTX file, error code: 102

### DISCUSSION THREAD

**Discussion by sherzlock** [infiniflow/ragflow#2461] [Q&A]
知识库是 general的解析，但里面有些文件是 pptx，所以单独设置为 presentation，但显示为 102，不支持
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/f31b3c24-b883-4c49-bbaa-537a0f53ceb1">

**Comment by KevinHuSh** [comment]
Fixed.
